doc/content/microreactors/gcmr/GCMR_Core_Neutronics.md

```diff
@@ -0,0 +1,100 @@
+# Neutronic Modeling of the Whole Core Gas-Cooled Microreactor (GCMR)
+
+## GCMR Core Description
+
+The GC-MR, developed at ANL, serves as a modeling experiment to explore design options considered by microreactor vendors, encompassing features like control drums, hydride metal, and TRISO fuel. This horizontal gas-cooled microreactor system boasts a thermal power of 20 MW and an approximate lifespan of 9.5 years. Its power conversion cycle utilizes a Brayton cycle, circulating high-temperature (650°C-850°C) and high-pressure (7 MPa) helium coolant. Surrounding the core are BeO radial and axial neutron reflectors, with twelve control drums positioned in the reflector encircling the core. These control rods, containing 96%-enriched B4C, are inserted into holes within the middle core assemblies. Displayed in [Fig_1] and [Fig_2]  are radial and axial views of the core, which is relatively compact, measuring 2.42 m in diameter and 2.40 m in length.
```

Comment:
I would say the GCMR model, developed at ANL,

Reply:
Added.